regression/pages/lms/dashboard_lms.py

```diff
@@ -42,3 +42,9 @@ def logout_lms(self):
             '.item a[href="/logout"]', 'SignOut button'
         )
         self.q(css='.item a[href="/logout"]').click()
+
+    def click_explore_courses_link(self):
+        """
+        Clicks Explore Courses link
+        """
+        self.q(css='.btn-neutral').click()
```

Comment:
if we are sure that this click will take us to one destination only we should add the wait for page here